modular_zubbers/code/modules/storyteller/gamemode.dm

```diff
@@ -211,8 +211,7 @@ SUBSYSTEM_DEF(gamemode)
 
 /// Gets the number of antagonists the antagonist injection events will stop rolling after.
 /datum/controller/subsystem/gamemode/proc/get_antag_cap()
-	var/cap = FLOOR((get_correct_popcount() / storyteller?.antag_divisor), 1) + sec_crew ? sec_crew : ANTAG_CAP_FLAT // Population divide by storyeteller's divisor, add one antag per sec.
-	return cap
+	return round(get_correct_popcount() / storyteller?.antag_divisor) + (sec_crew ? sec_crew : ANTAG_CAP_FLAT) // Population divide by storyeteller's divisor, add one antag per sec.
```

Comment:
> storyteller?
No- presumably this simply returns 0 if Storyteller is null. You'll need to either runtime or handle storyteller being null more cleanly above.

Reply:
Fair point actually, there is a very specific point where I saw this part runtime (Opening the game panel menu with no storyteller), since in any other case the procchain reaches this there has to be a storyteller